### DISCUSSION TITLE
PIMCORE 11: Building Grid Options - Search Functionality

### DISCUSSION THREAD

**Discussion by sandyHM** [pimcore/pimcore#17200] [Q&A / Support]
When building Grid options in PIM 11 the search is not filter out Classes & Containers that are not relevant. For example when I search SAP Description it should be a contains search that does not include items like Features and Marketing Data / Field Container, Field Container etc.
Step to reproduce:
1. open any folder that contain objects.
2.  when Search, Edit and Export screen is appear click on Grid Options.
3. try to search any attribute in Class Attributes
4. after entering more then 2 latter it show result with  empty Field Container. 

![grid option filter](https://github.com/pimcore/pimcore/assets/107986459/da2df3ba-7a37-432c-9a14-2122746daec8)

**Comment by sandyHM** [comment]
any update ?

**Comment by kingjia90** [comment]
Moved to https://github.com/pimcore/admin-ui-classic-bundle/issues/637
